src/com/rcv/ElectionConfig.java

```diff
@@ -57,11 +65,11 @@ public boolean validate() {
 
     // if multi-seat is indicated we validate decimal count and rules style
     //
-    if(this.numberOfWinners() > 1) {
-      if(this.decimalPlacesForVoteArithmetic() < 0 || this.decimalPlacesForVoteArithmetic() > 1000) {
+    if (this.numberOfWinners() > 1) {
+      if (this.decimalPlacesForVoteArithmetic() < 0 || this.decimalPlacesForVoteArithmetic() > 1000) {
```

Comment:
they'll probably want us to pull out 1000 into a constant

Reply:
yep, missed that.  Side note: eventually this whole validate function needs to be expanded to do range checking and provide meaningful output.